clustering/k-means-clustering-plugin/.gitignore

```diff
@@ -0,0 +1,175 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
All of the items in the `.gitignore` are in the base repository. This file should be removed.

Reply:
done